### PR TITLE
fix documentation typos and add documentation output + add TIME() function

### PR DESCRIPTION
**Pull request by MrEconomical**
e

### CODE REVIEW

**Review by MrEconomical — COMMENTED** [review]
@amasad please review these changes!

File: functions.js [hunk 1]
```diff
@@ -115,6 +115,10 @@ const Functions = {
   LOWERCASE(str) {
     return str.toLowerCase();
   },
+
+  TIME() {
+      return Date.now()
```
Comment (amasad) [inline functions.js]: use 2 spaces please

**Review by amasad — CHANGES_REQUESTED** [review]
use 2 spaces not 4.

Can you also update the docs.repl.it?

File: functions.js (hunk 1 above)
Comment (MrEconomical) [reply functions.js]: fixed

**Comment by MrEconomical** [comment]
synced both documentations here: https://github.com/replit/replit.github.io/pull/35
should be good now

**Comment by amasad** [comment]
Thank you `e`

**Pull request merged**